custom/enikshay/management/commands/investigations_reconciliation.py

```diff
@@ -184,26 +184,24 @@ def _get_open_person_case_ids_to_process():
                     print("processed %d / %d docs from db %s" % (i, num_case_ids, db))
 
 
-def get_all_occurrence_cases_from_person(person_case_id):
+def get_all_occurrence_case_ids_from_person(person_case_id):
     case_accessor = CaseAccessors(DOMAIN)
     all_cases = case_accessor.get_reverse_indexed_cases([person_case_id])
-    return [case for case in all_cases if case.type == CASE_TYPE_OCCURRENCE]
+    return [case.case_id for case in all_cases if case.type == CASE_TYPE_OCCURRENCE]
 
 
 def get_open_confirmed_drtb_episode_cases(person_case_id):
```

Comment:
There's an existing function `get_all_episode_cases_from_person` which does most of the work of these two functions (although it has the same issue you resolved in this PR, of not making use of a list of arguments).